mpa/modules/models/heads/custom_hierarchical_linear_cls_head.py

```diff
@@ -111,9 +111,9 @@ def simple_test(self, img):
         for i in range(self.hierarchical_info['num_multiclass_heads']):
             multiclass_logit = cls_score[:, self.hierarchical_info['head_idx_to_logits_range'][i][0]:
                                             self.hierarchical_info['head_idx_to_logits_range'][i][1]]
+            multiclass_logit = torch.softmax(multiclass_logit, dim=1)
```

Comment:
This is exactly the same way how predictions are done in d-o-r, so I think the changes are correct.